test/count.test.js

```diff
@@ -0,0 +1,62 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: loopback-connector-couchdb2
+// This file is licensed under the Apache License 2.0.
+// License text available at https://opensource.org/licenses/Apache-2.0
+
+'use strict';
+
+var _ = require('lodash');
+var should = require('should');
+var db, TestCountUser;
+
+if (!process.env.COUCHDB2_TEST_SKIP_INIT) {
+  require('./init.js');
+}
+
+function create50Samples() {
+  var r = [];
+  for (var i = 0; i < 70; i++) {
+    r.push({name: 'user'.concat(i)});
+  }
+  return r;
+};
+
+function cleanUpData(done) {
+  TestCountUser.destroyAll(done);
+};
+
+describe('count', function() {
+  before((done) => {
+    const config = _.assign(global.config, {globalLimit: 100});
+    const samples = create50Samples();
+    db = global.getDataSource(config);
+
+    TestCountUser = db.define('TestCountUser', {
+      name: {type: String},
+    }, {forceId: false});
+
+    db.automigrate((err) => {
+      if (err) return done(err);
+      TestCountUser.create(samples, done);
+    });
+  });
+
+  it('returns more than 25 results with global limit set', (done) => {
+    TestCountUser.count((err, r)=> {
+      if (err) return done(err);
+      console.log(r);
```

Comment:
Here we should check the size of the res to be 100.

Reply:
good catch!!

Reply:
I meant `COUNT_OF_SAMPLES` ;)